common-css/style-guide.css

```diff
@@ -0,0 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@600&family=Playfair+Display:wght@600&display=swap");
+
+html {
+  font-size: 62.5%;
+}
+/* colors */
+:root {
+  --primary: rgb(127, 36, 163);
+  --secondary: #8eecf5;
+  --text: #9c92a3;
+  --background: whitesmoke;
+  --heading: #221d23;
+  --gun-metal: #22333b;
+
+  /* typography */
+
+  /* font-size */
+
+  --f1: 0.8rem;
```

Comment:
This F1 S1 are confusing to read if you get time i can change them i guess

Reply:
Thanks Poornima for your review. will make the changes by next PR.